src/components/BingoBoardCard.vue

```diff
@@ -1,18 +1,19 @@
 <template>
   <v-card class="mt-16 mx-auto" elevation="10" width="850px">
     <p class="text-center my-2 text-h4" :style="titleStyles">Bingo Board</p>
-    <div v-if="show" class="text-center my-2 text-h4">
+    <div v-if="showRandomNumber" class="text-center my-2 text-h4">
       {{ bingo }}
     </div>
-    <v-card color="grey-lighten-4" class="mx-5 px-5 py-2 d-flex justify-center">
+    <div v-if="showWinner" class="text-center my-2 text-h4">
+      <p>Congratulations!! You are winner</p>
+    </div>
+    <v-card color="grey-lighten-4" class="mx-5 px-5 d-flex justify-center" flat>
       <div class="wrapper">
         <div
           class="box"
-          v-for="(number, index) in bingoNumbers"
+          v-for="(number, index) in bingoBoardNumbers"
           :key="index"
-          :style="{
-            backgroundColor: isBingo ? 'green' : 'transparent',
-          }"
+          :style="getColumnStyle(number, index)"

```

Comment:
Can use class bindings instead of style bindings. Inline styles shall be avoided.